src/db/migrations/20190728221040-create-downVote.js

```diff
@@ -0,0 +1,49 @@
+import Debug from 'debug';
+
+const debug = Debug('dev');
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    try {
+      await queryInterface.createTable('downVote', {
```

Comment:
rename file to `...-create-downvote` and then rename the table to `Downvotes`
```suggestion
      await queryInterface.createTable('Downvotes', {
```

Reply:
done